lighthouse-core/report/v2/renderer/dom.js

```diff
@@ -23,6 +23,30 @@ class DOM {
    */
   constructor(document) {
     this._document = document;
+
+    this._installDOMExtensions();
+  }
+
+  /**
+   * Add some API sugar
+   */
+  _installDOMExtensions() {
+    if (typeof self === 'undefined' || !self.Element) return;
+    if (self.Element.prototype.createChild) return;
+    const instance = this;
+
+    /**
+     * From devtools' DOMExtension.js (minus the 3rd customElementType param)
+     * @param {string} elementName
+     * @param {string=} className
+     * @return {!Element}
+     */
+    self.Element.prototype.createChild = function(elementName, className) {
+      const element = instance.createElement(elementName, className);
```

Comment:
i dont think we can because in devtools this method will already exist and it doesnt do attrs